products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,380 @@
+    team.name = "FC Barcelona"
+RETURN
+    max_fee, player_position, collect(p.name) as player_names
+```
+
+If we needed to get the maximum transfer fee per position we would only need first `MATCH` in the
+above query making it way shorter. In order to match which players had maximum transfer fees per position
+our query is split into two parts:
+* First `MATCH` in the query finds the maximum transfer fee per position.
+* Second `MATCH` in the query is finding all players transferred to "FC Barcelona" with the same position and transfer fee equal to the maximum one from the previous query.
+
+11) If you want to find all player transfers between two clubs you can do that also.
+
+```opencypher 
+MATCH 
+    (t:Transfer)<-[:TRANSFERRED_IN]-(player:Player)-[:TRANSFERRED_IN]->(:Transfer)<-[:TRANSFERRED_FROM]-(team:Team)
+WHERE
+    team.name = "FC Barcelona"
+WITH player, collect(t) as transfers
+MATCH 
+    player_path = (a:Team)-[*bfs..10 (e, n | 'Team' IN labels(n) OR ('Transfer' in labels(n) AND n in transfers) )]->(b:Team)
+WHERE
+    a.name = "FC Barcelona" AND
+    b.name = "Sevilla FC"
+UNWIND nodes(player_path) as player_path_node
+WITH player_path_node, player
+WHERE 'Team' in labels(player_path_node)
+WITH collect(player_path_node.name) as team_names, player
+RETURN player.name as player_name, team_names
+```
+In the above query, we will find all players that transferred from "FC Barcelona" to "Sevilla FC". It 
+will include direct transfers (from "FC Barcelona" to "Sevilla FC") and indirect transfers (from "FC Barcelona"
+to one or multiple other clubs and lastly "Sevilla FC"). That is the reason why we started first `MATCH` with
+searching for all players and transfers that were transferred from "FC Barcelona". Next up is the player transfer
+traversal through transfers and teams all the way to the "Sevilla FC".
+
+For this part, we used the breadth-first search (BFS) algorithm with lambda filter `(e, v | condition)`.
+It's a function that takes an edge symbol `e` and a vertex symbol `v` and decides whether this edge and vertex pair 
+should be considered valid in breadth-first expansion by returning true or false (or Null). In the above example,
+lambda is returning true if a vertex has a label `Team` or a label `Transfer`. If a vertex is `Transfer` there is an
+additional check where we need to make sure the transfer is one of the transfers of players transferred from "FC Barcelona".
+It needs to be either `Team` or `Transfer` because to get from a team that made the transfer to
+the team where the player is being transferred to, we need to go through the node `Transfer` that connects those two teams.
+So the traversal from "FC Barcelona" to "Sevilla FC" will go through the following nodes: Transfer, Team, Transfer, Team, Transfer, etc.
+
+
+12) In the previous query, we found all transfers between two clubs. Let's filter out direct ones now.
+Now we need small change in query to only get indirect transfers.
```

Comment:
```suggestion
We need to add a small change in the query to only get indirect transfers.
```
Just because the previous sentence ends with `Now`. So it is weird to read `... now. Now...`

Reply:
:+1: